testing-modules/rest-assured/src/main/java/com/baeldung/restassured/learner/Class.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.restassured.learner;
+
+class Class {
```

Comment:
We might want to name this "Course" instead of "Class", so as to not confuse it with java.lang.Class.